src/main/java/ru/naumen/servacc/ui/UIController.java

```diff
@@ -883,7 +855,6 @@ private void scheduleRefresh(final String filter, long delay)
         }
         TimerTask refreshTask = new TimerTask()
         {
-            @Override
             public void run()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)